### ISSUE TITLE
Design Intent - Accessibility Feedback - Employee Experience Team, 21-526EZ, Max CFI

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#70061]
## VFS actions
- [x] Assign this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [x] Comment on this ticket:
  - [ ] If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - [x] When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - [x] When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [x] Close the ticket when all feedback has been addressed.

## Thoughts/questions

-

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- **Must:** Echoing Erin in #69941, if you decide to link out to a separate information page, be sure to warn users about the new tab. I love visible text when the design allows for it, but an icon plus screen reader friendly text (either an `aria-label` or a `<span class="sr-only">`) is sufficient.
- **Should:** When placing the link/accordion/whichever element you ultimately decide on, I have a strong preference for it to come either before or after the full set of checkboxes, rather than pairing it with the checkbox. For screen reader users, navigating a fieldset of checkboxes goes like this:
   1. User tabs to the first checkbox.
   2. The question (eg. "Please choose the disability you are filing for...") is announced.
   3. The label for the first checkbox is announced, and the element is announced as a checkbox.
   4. User presses tab.
   5. The label for the second checkbox is announced, and the element is announced as a checkbox.
   6. Repeat 4 and 5 for each remaining checkbox.
   7. User tabs to the next element after the fieldset.
   
  When a user tabs to a link that's inside the fieldset, they might assume they've reached step 7 when they're actually halfway between 3 and 4. When the next tab stop is another checkbox --- and it's not announced with the question --- it may not be clear to them that they're still back under the umbrella of the question.
  
  Moving the link to either before or after the fieldset has some drawbacks, but I think preserving the mental model of how a list of checkboxes works is worthwhile.

**Comment by carodew** [comment]
Response: 

**MUST** 

We haven't coded this out yet, but I've made the following adaptions to the design file 
* added annotation regarding a11y requirements for engineers (See screenshot below) 
* added visual icon on link indicating it will open in a new tab

![Image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/10144074/65733358-5bc0-4d78-b924-f1eddcc35897)

**SHOULD**

* We're testing two designs in usability testing, both with the extra content at the _bottom_ of the list. [Sketch designs, see "Platform team feedback" page](https://sketch.com/s/4319cad9-d81b-4b8e-8339-d4150fbd5274)
